mdpow/workflows/dihedrals.py

```diff
@@ -157,18 +167,35 @@ def rdkit_conversion(u, resname):
 
     """
 
-    try:
-        solute = u.select_atoms(f'resname {resname}')
-        mol = solute.convert_to('RDKIT')
-    except AttributeError:
-        elements = [guess_atom_element(name) for name in u.atoms.names]
-        u.add_TopologyAttr("elements", elements)
-        solute = u.select_atoms(f'resname {resname}')
-        mol = solute.convert_to('RDKIT')
+    # i think somewhere we decided to ditch try/except method, will check notes
+    # notes pertaining to testing topology attributes for explicit Hs
+    
+    #try:
+    #    solute = u.select_atoms(f'resname {resname}')
+    #    mol = solute.convert_to('RDKIT')
+    #except AttributeError:
+    #    elements = [guess_atom_element(name) for name in u.atoms.names]
+    #    u.add_TopologyAttr("elements", elements)
+    #    solute = u.select_atoms(f'resname {resname}')
+    #    mol = solute.convert_to('RDKIT')
+
+    elements = [guess_atom_element(name) for name in u.atoms.names]
+    u.add_TopologyAttr("elements", elements)
+
+    # com or cog?
+    solute = u.select_atoms(f'resname {resname}')
+    solute.unwrap(compound='residues', reference='com')
+
+    mol = solute.convert_to('RDKIT')
+    rdCoordGen.AddCoords(mol)
+    #rdDepictor.Compute2DCoords(mol), not sure how this is different from above^
+
+    for atom in mol.GetAtoms():
+        atom.SetProp("atomNote", str(atom.GetIdx()))
```

Comment:
State in the docs what we are attaching as the label (is it the 0-based or 1-based atom index... I already forgot... so that's why docs are needed ;-) )